bitnami/kafka/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kafka
-version: 11.6.6
+version: 11.8.0
```

Comment:
Did you skip `11.7.0`?

Reply:
I made a mistake into the last commit, fixing